ow_lander/src/ow_lander/constants.py

```diff
@@ -5,12 +5,12 @@
 """Define constants required by multiple modules within the package"""
 
 ## GLOBAL VARS ##
-J_SCOOP_YAW = 5
-J_HAND_YAW = 4
-J_DIST_PITCH = 3
-J_PROX_PITCH = 2
-J_SHOU_PITCH = 1
 J_SHOU_YAW = 0
+J_SHOU_PITCH = 1
+J_PROX_PITCH = 2
+J_DIST_PITCH = 3
+J_HAND_YAW = 4
+J_SCOOP_YAW = 5
```

Comment:
Should J_GRINDER also be 5? If so, you should probably add a comment to explain why they are the same value.

Reply:
> I'm not a fan of this notation. Private should be absolute. Using __really_private implies that _private has been demoted to just a suggestion. But I don't hate it enough to hold up this PR.

There is no real way to enforce private members in Python, but `_` and `__` partially serve that function. `_` is purely notionally, and indicates to a programer that this value/method should not be touched outside of the class its in or its subclasses. `__` mangles the name of the member in a way that make it still possible to access, but requires both its name and the name of the class its defined in. 

I have read (not in the ROS Py style guide) that the difference between `_` and `__` class member prefixes are analogous to protected and private members in C++. So anything you expect only a subclass to reference is prefixed with `_`, and anything entirely private to its class should be prefixed `__`. 

I have to admit I haven't been entirely rigorous with this in the ow_lander package.

**Source** https://stackoverflow.com/questions/1301346/what-is-the-meaning-of-single-and-double-underscore-before-an-object-name